ci-operator/step-registry/telco5g/cluster-setup/telco5g-cluster-setup-commands.sh

```diff
@@ -49,13 +52,24 @@ EOF
 ADDITIONAL_ARG=""
 # default to the first cluster in the array, unless 4.14
```

Comment:
"# default to the first cluster in the array, unless 4.14"
- what is the reasoning behind this?

Reply:
https://issues.redhat.com/browse/CNF-7848